nixos/modules/services/misc/portunus.nix

```diff
@@ -22,7 +22,7 @@ in
       description = lib.mdDoc ''
         Port where the Portunus webserver should listen on.
 
-        This must be put behind a TLS-capable reverse proxy because Portunus only listens on localhost.
+        This must be put behind a TLS-capable reverse proxy because Portunus is only configured to listen on ::1, the IPv6 loopback address.
```

Comment:
Portunus actually listens on dualstack but I don't think we need that change and I already opened the other as #221684

Reply:
If you look at my table in https://github.com/NixOS/nixpkgs/issues/221678, it isn't dualstack (if I understand correctly). Apparently [::1] is either IPv4, or IPv6. It is true that [::] was both, but we have removed that.